src/rest.ts

```diff
@@ -39,18 +40,29 @@ export function getTDPDatabases(): Promise<IDatabaseDesc[]> {
   return getAPIJSON(`${REST_DB_NAMESPACE}/`);
 }
 
-export interface IViewDesc {
+export interface IViewDescription {
```

Comment:
```suggestion
export interface IViewColumnDesc {
```

Reply:
Sounds more like it is a description of the views columns, imo.

Reply:
Mh, true. Our rationale was to unify it to `...Desc`, which was not possible in this case, because `IViewDesc` is already in use. So we came up with this interface name. Do you have a different suggestion?

Reply:
the rationale behind the naming @thinkh proposed was that we wanted to rename it to have the `Desc` suffix that it is consistent with the other names. and we cannot name it IViewDesc, beacuse that was the original name of `IDatabaseViewDesc`

I'd say we either name it `IServerColumnDesc`, because it contains an `IServerColumn[]` + and IDType as meta data

Reply:
Now that I think about it, I guess both is fine (`IViewColumnDesc`/`IServerColumnDesc`)  because the interface is no longer used to describe a view but a set of columns.

Reply:
Then let's go for `IServerColumnDesc`. Could you please change this?

Reply:
Done